js/script.js

```diff
@@ -0,0 +1,48 @@
+var modals = document.getElementsByClassName('modal-bg');
+var links = document.getElementsByClassName('nav__link');
+var closeButtons = document.getElementsByClassName("modal--close")
+var choiceOptions = document.getElementsByClassName('choice-container__icon');
+var userChoice;
+var openedModal;
+
+
+for(var i=0; i < links.length; i++) {
+    links[i].addEventListener("click", showModal);
+    links[i].elementNumber = i;
+    closeButtons[i].addEventListener("click", closeModal);
+    window.addEventListener("click", closeModalWhenClickOutside);
+}
+
+function showModal(e) {
+    if(openedModal != null) {
+        closeModal();
+}
+    openedModal = e.srcElement.elementNumber;    
+    modals[openedModal].style.visibility = 'visible';
+};
+
+function closeModal() {
+    modals[openedModal].style.visibility = 'hidden';
+};
+
+function closeModalWhenClickOutside(e) {
+    if(e.target == modals[openedModal]) {
```

Comment:
Doesn't this statement mean that when the opened modal is the target of a click event, it should be closed?